### ISSUE TITLE
bug: break any page

### ISSUE THREAD

**Issue by workaj** [HoltBosse/Alba#198]
https://github.com/HoltBosse/Alba/commit/ce9f1174cc554a3b272c0beaa765bb156a75d266#diff-48b65cf7f112f733c3d3240994f6aa664e943fab2fb1617be73874f205fc9091R629-R635 breaks any page that does not have a controller assigned to it. this will break several sites. in addition, the commit is labeled `better homepage 404 testing`, however it introduces some half finished caching method this is broken since core/cache.php does not exist. 

tldr: revert https://github.com/HoltBosse/Alba/commit/ce9f1174cc554a3b272c0beaa765bb156a75d266#diff-48b65cf7f112f733c3d3240994f6aa664e943fab2fb1617be73874f205fc9091 as it breaks many things
